File: server/internal/constants/order_status.go
package constants

type OrderStatus string

const (
	OrderStatusCreated         OrderStatus = "CREATED"
	OrderStatusConfirmed       OrderStatus = "CONFIRMED"
	OrderStatusInDeliveryRoute OrderStatus = "IN_DELIVERY_ROUTE"
	OrderStatusFinished        OrderStatus = "FINISHED"
	OrderStatusCanceled        OrderStatus = "CANCELED"
)

File: web/src/enums/order_status.ts
export const OrderStatus = {
  CREATED: 'CREATED',
  CONFIRMED: 'CONFIRMED',
  IN_DELIVERY_ROUTE: 'IN_DELIVERY_ROUTE',
  FINISHED: 'FINISHED',
  CANCELED: 'CANCELED'
} as const;

export type OrderStatus = (typeof OrderStatus)[keyof typeof OrderStatus];
